website/src/home/content/code-of-conduct.md

```diff
@@ -0,0 +1,55 @@
+# Code of Conduct
+
+Compa aims to provide resources to students across the world. Diversity is one of our strengths and we want to ensure that everyone feels welcome and included. This code applies to everyone in our community, across all platforms.
```

Comment:
This opening remarks feels off for me. "provide resource to students across the world", "diversity", feels like a stretch right now.

Maybe:

Compa is an open platform for collaborating with other students. To ensure information shared are correct, respect each other on the platform and keep this initiative going, we suggest you follow our code of conduct.

What do you think? Maybe edit along those lines?

Reply:
yeah, i didn't have any strong ideas at the time, so i went with generic. take a look at this change

Reply:
Yupp I like that. Thanks